modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2370,10 +2371,17 @@
 	end
 
 
-
 	function m.warningLevel(cfg)
 		local map = { Off = "TurnOffAllWarnings", Extra = "Level4" }
-		m.element("WarningLevel", nil, "%s", map[cfg.warnings] or "Level3")
+		m.element("WarningLevel", nil, map[cfg.warnings] or "Level3")
+	end
+
+
+	function m.warningLevelFile(cfg, condition)
```

Comment:
Could this be merged with `m.warningLevel`? I'm pretty sure the other file-level functions do that?

Reply:
This commit is a squash of 3 commits that tried that exact approach, and failed, so after going though those iterations, this is what ultimately worked.